scripts/docs_config/glossary.rst

```diff
@@ -0,0 +1,97 @@
+Glossary
+==========================================================
+
+Homogeneous Memory
+  A collection of memory composed of a single memory type, managed by a singular 
```

Comment:
This sentence sounds weird to me. Can we say 'a collection of memory'? 
Also, what is a 'memory type' (are you referring to the underlying technology?)

It may be better to define a 'Homogenous Memory System' instead of just 'Homogenous Memory'.